jobs/integr8ly/pds-install.yaml

```diff
@@ -168,9 +168,12 @@
                         dir('installation/evals') {
                             
                             String ansibleParams = ''
-                            if(GH_CLIENT_ID && GH_CLIENT_SECRET) {
+                            if (GH_CLIENT_ID && GH_CLIENT_SECRET) {
                                 ansibleParams = "-e github_client_id=${GH_CLIENT_ID} -e github_client_secret=${GH_CLIENT_SECRET}"
                             }
+                            if (ADDITIONAL_ANSIBLE_PARAMS.indexOf('eval_seed_users_count') == -1) {
```

Comment:
Haven't really tried, but ADDITIONAL_ANSIBLE_PARAM has no default value. If you just left it empty won't you get an undefined error?

Reply:
It will not fail. See this test pipeline's output (one build with provided param and one without): https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/ppaszki-test/. The empty param will be treated as an empty string

Reply:
@trepel can you please have a final look?

Reply:
Hey, I am sorry. I should have just googled this out. Good to know.